src/components/app.tsx

```diff
@@ -919,9 +1070,10 @@ export class App extends React.Component<AppProps, AppState> {
                     onNewRun={this.checkNewData}
                     onSaveData={this.sendData}
                     onReloadPage={this.reload}
+                    onAboutClick={this.showAbout}
+                    isDisabled={sensorManager===null}
```

Comment:
```suggestion
                    isDisabled={sensorManager == null}
```
in case `sensorManager === undefined`.

Reply:
Fixed.